iocore/net/YamlSNIConfig.cc

```diff
@@ -213,6 +213,38 @@ std::set<std::string> valid_sni_config_keys = {TS_fqdn,
 namespace YAML
 {
 template <> struct convert<YamlSNIConfig::Item> {
+  static ts::port_range_t
+  parse_single_inbound_port_range(Node const &node, swoc::TextView port_view)
+  {
+    auto min{port_view.split_prefix_at('-')};
+    if (!min) {
+      min = port_view;
+    }
+    auto const &max{port_view};
```

Comment:
Don't use references with views - they are essentially references already.

Reply:
Thanks for catching this. I've caught myself doing this in other places and am trying to develop the new habit.